src/RMP/Translate/lang/programmes/cy.po

```diff
@@ -13,7 +13,10 @@ msgstr "Am BBC Radio"
 
 #. English: "Add stuff to listen later. And follow your favourite shows to get updates and recommendations. All on any device.".
 msgid "add_stuff_listen_later"
-msgstr "Ychwanegwch gynnwys er mwyn gwrando nes ymlaen. A dilynwch eich hoff raglenni i gael y wybodaeth a’r argymhellion diweddaraf. Pob dim ar unrhyw ddyfais."
+msgstr ""
```

Comment:
This looks a bit dodgy @steveculverwell 

Looks like it's happened in the other files too